src/coreclr/vm/jitinterface.cpp

```diff
@@ -13911,50 +13854,7 @@ BOOL LoadDynamicInfoEntry(Module *currentModule,
         }
         break;
 
-#if defined(PROFILING_SUPPORTED)
```

Comment:
This is a different kind of profiling support?

Reply:
This was a left-over from the special NGen image flavor with prolog/epilog profiler callouts. We do not have those anymore. If the profiler wants callouts, we fallback to JIT.